Snowflake.Data/Core/SFSessionProperty.cs

```diff
@@ -170,6 +170,35 @@ internal static SFSessionProperties parseConnectionString(String connectionStrin
                 }
             }
 
+            //handle DbConnectionStringBuilder missing cases

```

Comment:
We need to investigate why DbConnectionStringBuilder missing this and we need to hand code this stuff.

Reply:
DbConnectionStringBuilder is good for password and special characters. It doesn't handle using double quote to warp the database name means case sensitive.